pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,70 +1,125 @@
+
+   Alternatively, you can install the latest release protocol by modifying [this curl command](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8) for your case.
+
+3. Extract the binary.
+
+   Extract the `.tar.gz` file that you downloaded. To run `dydxprotocold` from the command line, either rename the extracted file or create a symbolic link to the file:
+
+   **Option 1:** Rename the file. Edit the filename from `dydxprotocold-<version>-<architecture>` to simply `dydxprotocold`. You might need to add your current directory to your $PATH or move the file to a directory in your $PATH.
+
+   **Option 2:** Create a symbolic link to the file using the name `dydxprotocold`. Run the following command:
+   ```bash
+   ln -s /path/to/your/binary dydxprotocold
+   ```
+
+4. Initialize your data directory.
+
+   First, make sure that your data directory is empty. In the example below, `DYDX_HOME` contains the path to the directory that must be empty.
+
+   Then, run the `dydxprotocold init` command, supplying a chain ID, a path to a data directory, and a moniker for your node:
+
+   ```bash
+   # Example values
+   CHAIN_ID=my-dydx-deployment
+   DYDX_HOME=/path/to/your/data/directory
+   NODE_MONIKER=my-dydx-fullnode
+   
+   dydxprotocold init --chain-id=$CHAIN_ID --home=$DYDX_HOME $NODE_MONIKER
+   ```
+
+After you initialize your data directory, your full node can write to it.
+
+## Install a snapshot of the dYdX chain's history
+Installing a snapshot saves time by syncing your full node to the history of the dYdX chain. This avoids downloading and validating the entire blockchain.
+
+1. Download the latest snapshot contents from https://bwarelabs.com/snapshots/dydx. 
+
+   If you can’t download the snapshot contents from Bware, you can download the snapshot contents from the following alternative sources:
+   - https://polkachu.com/tendermint_snapshots/dydx
+   - https://dydx-archive-snapshot.kingnodes.com
+   - Also check [Snapshot service](/infrastructure_providers-network/resources#snapshot-service)
+
+2. Extract the snapshot to your data directory.
+
+   In your data directory, run the following command using your own snapshot filename:
+   ```bash
+   # Example value
+   SNAPSHOT_FILENAME=dydx2024example
+
+   lz4 -dc < $SNAPSHOT_FILENAME.tar.lz4 | tar xf -
+   ```
+
+When you start your full node, it will automatically use the snapshot you saved to its data directory.
+
+## Start your full node
+
+Configuring and starting your full node for the first time allows it to sync with the dYdX chain network. If you saved a dYdX snapshot to your data directory, starting your node will first use that snapshot to quickly recreate most of the chain's history.
+
+1. Configure parameters in your command line. Use the following syntax:
+
+   ```bash
+   # Example values
+   SEED_LIST="123@seeds.polkachu.com:123,123@seeds.kingnodes.net:123"
+   DYDX_HOME=/path/to/your/data/directory
```

Comment:
I think we should leave it for quick reference since it's a variable in that example as well and users may be referencing specific steps. It could be confusing if they need to scroll back up to find that info